src/i18n/de/ui.ts

```diff
@@ -58,16 +71,54 @@ export default UIDictionary({
 	'aside.tip': 'Tipp',
 	'aside.caution': 'Achtung',
 	'aside.danger': 'Gefahr',
+	// `<LanguageSelect>` vocabulary
+	'languageSelect.label': 'Sprache auswählen',
+  	// Integrations vocabulary
+	'integrations.changelog': 'Changelog',
+	'integrations.footerTitle': 'Weitere Integrationen',
+	'integrations.renderers': 'UI-Frameworks',
+	'integrations.adapters': 'SSR-Adapter',
+	'integrations.others': 'Sonstiges',
+  	// Checklist component
+	'checklist.or': 'oder',
+  	// Multiple Choice component
+	'multipleChoice.defaultCorrect': 'Korrekt!',
+  	'multipleChoice.defaultIncorrect': "Versuch's nochmal!",
+  	'multipleChoice.submitLabel': 'Absenden',
+  	// Tutorial Progress
+	'progress.todo': 'To-do',
+  	'progress.done': 'Erledigt',
+  	// Tutorial Navigation
+  	'tutorial.trackerLabel': 'Tutorial Tracker',
+	'tutorial.unit': 'Teil',
+  	// Tutorial
+	'tutorial.getReady': 'Mach dich bereit,…',
 	// Feedback Fish widget
 	'feedback.button': 'Gib uns Feedback',
 	'feedback.a11yLabel': 'Feedback-Formular',
 	'feedback.formTitle': 'Wie kann ich helfen?',
 	'feedback.categoryGroupLabel': 'Feedback-Typ wählen',
 	'feedback.issue': 'Problem',
+	'feedback.createIssue': 'Erstelle Github-Problem',
```

Comment:
```suggestion
	'feedback.createIssue': 'Issue auf GitHub erstellen',
```

Wird auf einem Button im Rahmen einer Entweder/Oder-Auswahl angezeigt (die Alternative ist "Feedback abschicken"). Habe die Formulierung daher angeglichen und auch GitHub korrekt geschrieben.

Man könnte auch "GitHub-Issue erstellen" oder "GitHub-Problem erstellen" nehmen, aber ich fand diese Version klarer. Passt scheinbar auch gerade noch so auf den Button, ohne zweizeilig zu werden. :)